XmlFormatter/src/Settings/Provider/DataStructure/SerializableSetting.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace XmlFormatter.src.Settings.Provider.DataStructure
+{
+    /// <summary>
+    /// This class is a data container to for the xml saving
+    /// </summary>
+    public class SerializableSetting
+    {
+        /// <summary>
+        /// Public access to hey key/name of this setting
+        /// </summary>
+        public string Name;
+
+        /// <summary>
+        /// Public access to the data string the
+        /// </summary>
+        public string Value;
+
+        /// <summary>
+        /// Public access to the type the value should be castet to
+        /// </summary>
+        public string Type;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5053335)